### DISCUSSION TITLE
How to manually insert chromosome sizes file in development version

### DISCUSSION THREAD

**Discussion by ytrink** [aertslab/scenicplus#344] [Q&A]
Hi, thanks for this great tool.
Apologies if this is a very simple question as I am learning snakemake on the fly. 
My university server blocks downloading from certain websites, so my run times out during the "download_genome_annotations" rule.

I see there is an option to manually insert chromosome sizes. I tried to do this by deleting the download_genome_annotation rule and instead providing my own chromosome sizes and gene annotations file (I saw you did this in the pycistarget vignette) in the "get_search_space" rule as input files.
However I keep running into key errors for the gene annotations file; either "Gene" or "Start" is not found. This despite the fact that read_table() returns a dataframe with these columns.

TLDR would it be possible to outline the steps necessary to skip any downloading steps and manually insert my own files?
Thanks very much,
Yaron
